radixdlt/src/test/java/org/radix/network/discovery/BootstrapDiscoveryTest.java

```diff
@@ -114,71 +83,90 @@ public void tearDown() {
         doReturn(new ByteArrayInputStream(expected.getBytes(StandardCharsets.US_ASCII))).when(conn).getInputStream();
-        assertEquals(expected,  getNextNode.invoke(new BootstrapDiscovery(config, universe), url));
+        assertEquals(expected, new BootstrapDiscovery(config, universe).getNextNode(url));
     }
 
     @Test
-	public void testGetNextNode_RuntimeException() throws IOException, ReflectiveOperationException
-	{
+	public void testGetNextNode_RuntimeException() throws IOException {
         doThrow(new RuntimeException()).when(conn).connect();
-        assertEquals(null,  getNextNode.invoke(new BootstrapDiscovery(config, universe), url));
+        assertNull(new BootstrapDiscovery(config, universe).getNextNode(url));
     }
 
-    @Test
-	public void testGetNextNode_InterruptedException() throws ReflectiveOperationException, InterruptedException
-	{
+	public void testGetNextNode_InterruptedException() throws InterruptedException {
         doThrow(new InterruptedException()).when(Thread.class);
         Thread.sleep(anyLong());
-        assertEquals(null,  getNextNode.invoke(new BootstrapDiscovery(config, universe), url));
+        assertNull(new BootstrapDiscovery(config, universe).getNextNode(url));
     }
 
     @Test
-	public void testConstructor_Seeds() throws Exception
-	{
+	public void testConstructor_Seeds() {
         doReturn("").when(config).get("network.discovery.urls", "");
         doReturn("1.1.1.1").when(config).get("network.seeds", "");
-        doReturn(8).when(config).get(eq("network.connections.in"), anyInt());
-        doReturn(8).when(config).get(eq("network.connections.out"), anyInt());
-        doReturn(8000).when(config).get(eq("messaging.inbound.queue_max"), anyInt());
-        doReturn(8000).when(config).get(eq("messaging.outbound.queue_max"), anyInt());
-        doReturn(30).when(config).get(eq("messaging.time_to_live"), anyInt());
-        BootstrapDiscovery testSubject = Whitebox.invokeConstructor(BootstrapDiscovery.class, config, universe);
+        BootstrapDiscovery testSubject = new BootstrapDiscovery(config, universe);
         Set<?> hosts = Whitebox.getInternalState(testSubject, "hosts");
         assertEquals(1, hosts.size());
     }
 
-    @Test
-	public void testConstructor_NeedHTTPS() throws Exception
-	{
-        thrown.expect(IllegalStateException.class);
+    @Test(expected = IllegalStateException.class)
+	public void testConstructor_NeedHTTPS() {
         doReturn("http://example.com").when(config).get("network.discovery.urls", "");
-        Whitebox.invokeConstructor(BootstrapDiscovery.class, config, universe);
+        assertNotNull(new BootstrapDiscovery(config, universe));
+    }
+
+    @Test
+    public void testDiscoverNoFilter() throws UnknownHostException {
+        doReturn("").when(config).get("network.discovery.urls", "");
+        doReturn("1.1.1.1").when(config).get("network.seeds", "");
+    	BootstrapDiscovery discovery = new BootstrapDiscovery(config, universe);
```

Comment:
Weird formatting (looks like 2 spaces instead of tabs)

Reply:
Will convert all spaces -> tabs here